comp2.html

```diff
@@ -0,0 +1,180 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+        <link rel="stylesheet" href="css/normalize.min.css">
+        <link rel="stylesheet" href="css/main.css">
+
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <div class="main">
+          <div class="sidebar">
+            <div class="logo-motorpool">
+              <p class="logo-title visuallyhidden">Motorpool</p>
+            </div>
+
+            <form class="search-form">
+              <input type="text" size=26 placeholder= "Search..."/>
+
+              <button type="submit">
+                <span class="visuallyhidden">Submit</span>
+              </button>
+            </form>
+
+            <ul class="events">
+              <li class="event">
+                <a href="#">[2014] TTT: Kings Dominion</a>
+              </li>
+              <li class="event">
+                <a href="#">[2013] TTT: Paintballing</a>
+              </li>
+              <li class="event">
+                <a href="#">[2014] TTT: GoKart Racing</a>
+              </li>
+              <li class="event">
+                <a href="#">[2013] TTT: Montfair</a>
+              </li>
+            </ul>
+
+            <div class="sidebar-actions">
+              <a href="#" class="new-event">
+                <span class="add-icon">Create New Event</span>
+              </a>
+            </div>
+          </div>
+
+          <div class="content">
+            <div class="content-heading">
+              <div class="heading-title">TTT: Kings Dominion</div>
+              <a href="#" class="btn btn-primary">Edit</a>
+            </div>
+
+            <hr>
+
+            <form class="route-form">
+              <div class="route-form-header">
+                <div class="route-form-title">
+                  [HQ] <span class="black">&harr;</span> Meadow Event Park
+                </div>
+
+                <ul class="route-form-actions">
+                  <li class="route-form-action">
+                    <a href="#" class="btn-link btn-small btn-secondary">Cancel</a>
```

Comment:
oh, point taken. in that case, maybe "#TODO" the href so it doesn't look like it's just supposed to be a dead link :+1: